app/Http/Controllers/Api/Trips/DeleteController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Api\Trips;
+
+use App\Domain\Trips\Models\Trip;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Response;
+
+final class DeleteController extends Controller
+{
+    public function __invoke(Trip $trip): JsonResponse
+    {
```

Comment:
Probably worth discussing as a standard but I feel its better to handle the authorisation checks as a middleware declaration on the route e.g. https://laravel.com/docs/10.x/authorization#via-middleware

Reply:
My reasoning for personal choice is you get a good top level overview of which routes are covered by authorisation rules.